packages/block-template/package.json

```diff
@@ -26,7 +26,8 @@
     "dev": "concurrently -n webpack,webpack-dev-server -c green,cyan \"yarn build:dev --watch\" \"yarn run-dev-server\"",
     "lint:tsc": "tsc --noEmit",
     "run-dev-server": "cross-env NODE_ENV=development webpack-dev-server --config webpack-dev-server.config.js",
-    "schema": "typescript-json-schema tsconfig.json AppProps --required true --out dist/block-schema.json"
+    "schema": "typescript-json-schema tsconfig.json AppProps --required true --out dist/block-schema.json",
+    "serve": "serve -C -l 61234 ./dist/"
```

Comment:
Not sure we need this dependency and the script in each new block. Instead, we can change instructions to use `npx serve dist` – it will work for everyone regardless of what’s in `yarn.lock`.

Adding `serve` dependency pins the version and the port. This means ‘you need to upgrade’ warnings sooner or later. Hardcoding the port would make it more difficult to manage multiple local blocks.

I was actually thinking of removing `serve` from all HASH blocks and running `npx serve` in the blocks directory. Local block paths would be something like `localhost:500/paragraph/dist/`, `localhost:500/header/dist/` etc.

Reply:
So maybe we just need one of these commands in docs:

```
npx serve --cors --listen 1234 dist
npx serve --cors dist

npx serve -C -l 1234 dist
npx serve -C dist
```

The port will be pinned to 5000 if we omit the option.

Reply:
Why not alias it in as a script?

Reply:
I've updated to remove `serve` as a dependency. I've kept a specified port because in practice the default is often going to be in use (it recently [changed to 3000](https://github.com/vercel/serve/pull/680)), and the fallback changes on each run. This means that it is harder to use as a source for a block in HASH app as it will keep changing where it's served from.

Reply:
Oh I forgot about `serve`’s shift from port 5000 to port 3000. I [complained](https://github.com/vercel/serve/pull/680#issuecomment-959812165) about it, but this did not change their decision back in the day 😅